### PR TITLE
[FEATURE] Search

### LINKED ISSUE #7: Página de Notificações

### LINKED ISSUE #68: Busca de tecnologias e serviços
Integração com Algolia para busca

### PR DESCRIPTION
**Pull request by davioliveiira**
## Summary

Resolves #68 

simulator_screenshot_653704AC-043C-4678-81F7-D3816217A6EB

### CODE REVIEW

**Review by mtullyoc — COMMENTED** [review]
LGTM! 

|,,|_

File: src/screens/Search/components/List/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  //         >
+  //           <Text>{item.label}</Text>
+  //           <View>
+  //             <Text>{item.count}</Text>
+  //           </View>
+  //         </TouchableOpacity>
+  //       ))}
+  //     </View>
+  //   </View>
+  // );
```
Comment (mateus4k) [inline src/screens/Search/components/List/index.tsx:51]: 👀

Comment (mateus4k) [inline src/screens/Search/components/List/index.tsx]: 👀

File: src/screens/Search/components/List/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Text, View, FlatList } from 'react-native';
+// import { View, Text, TouchableOpacity } from 'react-native';
```
Comment (mateus4k) [inline src/screens/Search/components/List/index.tsx]:
```suggestion
```

File: src/screens/Search/components/List/index.tsx (hunk 1 above)
Comment (davioliveiira) [reply src/screens/Search/components/List/index.tsx]: Está em WIP ainda @mateus4k hehehe!

File: src/screens/Search/components/List/index.tsx (hunk 2 above)
Comment (davioliveiira) [reply src/screens/Search/components/List/index.tsx]: Está em WIP ainda @mateus4k hehehe!

Comment (mateus4k) [reply src/screens/Search/components/List/index.tsx]: Foi mal kkkk

File: src/screens/Search/index.tsx [hunk 3]
```diff
@@ -29,32 +34,42 @@ const SearchBox = connectSearchBox((props: any): JSX.Element => {
...
-    </InstantSearch>
-  </S.Wrapper>
-);
+const Search = (): JSX.Element => {
+  const [tab, setTab] = useState<number>(1);
+  const indexSearch = [ALGOLIA_INDEX_NAME_TECHNOLOGY, ALGOLIA_INDEX_NAME_SERVICE];
+
+  const onChange = (idx: number) => setTab(idx);
+
+  // useEffect(() => {}, [tab]);
```
Comment (mtullyoc) [inline src/screens/Search/index.tsx]:
```suggestion

```

**Review by mtullyoc — CHANGES_REQUESTED** [review]
Good job! 
:)

**Review by mtullyoc — APPROVED** [review]

File: src/screens/Search/index.tsx (hunk 3 above)
Comment (mateus4k) [inline src/screens/Search/index.tsx]:
```suggestion
```

File: .env.example [hunk 4]
```diff
@@ -1,4 +1,5 @@
 API_URL='http://192.168.1.3:3333'
 ALGOLIA_APP_ID=GC7K0ETHXB
 ALGOLIA_ADMIN_KEY=8ea4ffa0588206bb0c0751294097e875
-ALGOLIA_INDEX_NAME=searchable_data_staging
\ No newline at end of file
+ALGOLIA_INDEX_NAME_TECHNOLOGY=searchable_data_staging
+ALGOLIA_INDEX_NAME_SERVICE=searchable_service_staging
```
Comment (mateus4k) [inline .env.example]:
```suggestion
ALGOLIA_INDEX_NAME_TECHNOLOGY=searchable_technology_staging
ALGOLIA_INDEX_NAME_SERVICE=searchable_service_staging
```
Anyway, it's better define this variables like `APP_ENV:staging` and `ALGOLIA_PREFIX:searchable` and generate the index name dynamically from "prefix + entity name (like `technology` or `service`) + ambient sufix 🤔

**Pull request merged**